### PR TITLE
Axpd-7405: Add Support for Schema Version Resource CRD

### PR DESCRIPTION
**Pull request by danny-akp**
- Schema version is a resource that supports CREAT, READ and DELETE

### CODE REVIEW

File: axual-webclient/client.go [hunk 1]
```diff
@@ -91,6 +93,7 @@ func (c *Client) RequestAndMap(method string, url string, reqBody io.Reader, hea
 		return err
 	}
 
+	log.Println("Logging")
```
Comment (kasparAxual) [inline axual-webclient/client.go:96]: Please remove this

**Review by kasparAxual — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 ## [master] - TBR
 * Support Managing Environments
 * Support Authorizing Application Access Grants in Terraform.
+* Support Managing Schema Version
```
Comment (markaxual) [inline CHANGELOG.md:9]: Maybe we can say Support Managing Avro Schemas

File: axual-webclient/schema_version_data.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
+package webclient
```
Comment (markaxual) [inline axual-webclient/schema_version_data.go:1]: Let's keep formatting similar to existing files. Your spacing is inconsistent

File: internal/provider/provider.go [hunk 4]
```diff
@@ -132,6 +132,7 @@ func (p *provider) GetResources(_ context.Context) (map[string]tfsdk.ResourceTyp
 		"axual_application_access_grant":           applicationAccessGrantResourceType{},
 		"axual_application_access_grant_approval":  applicationAccessGrantApprovalResourceType{},
 		"axual_application_access_grant_rejection": applicationAccessGrantRejectionResourceType{},
+		"axual_schema_version": schemaVersionResourceType{},
```
Comment (markaxual) [inline internal/provider/provider.go:135]: Fix Formatting

File: internal/provider/resource_schema_version.go [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+			},
+			"description": {
+				MarkdownDescription: "A short text describing the Schema version",
+				Optional:            true,
+				Type:                types.StringType,
+				Validators: []tfsdk.AttributeValidator{
+					validation.Length(0, 500),
+				},
+			},
+			"schema_version_id": {
```
Comment (markaxual) [inline internal/provider/resource_schema_version.go:49]: Can we keep this as id. I would prefer it this way as it's consistent with all our other resources.

**Review by markaxual — CHANGES_REQUESTED** [review]

**Review by markaxual — APPROVED** [review]

**Review by yunusaxual — APPROVED** [review]

**Pull request merged**